README.md

```diff
@@ -93,3 +93,27 @@ to install qXmoji to the selected prefix and DESTDIR.
 If your default `make` tool is not GNU make, the latter is probably installed
 using the name `gmake` (e.g. on BSD systems), so type that instead of `make`.
 
+## Building on Linux (Debian / Ubuntu)
+
+Clone the current source tree with the needed submodules.
+
+    git clone git@github.com:jhx0/qxmoji.git --recurse-submodules
```

Comment:
Cloning should not be platform specific. Also, the standard method is https. I suggest to leave this out